src/renderer/services/plugin-manager.js

```diff
@@ -368,6 +369,38 @@ class PluginManager {
     }
   }
 
+  async loadThemes (pluginObject, plugin, profileId) {
+    if (!pluginObject.hasOwnProperty('getThemes')) {
+      return
+    }
+
+    const pluginThemes = this.normalize(pluginObject.getThemes())
+    if (pluginThemes && isObject(pluginThemes)) {
+      // Validate the configuration of each theme and ensure that their CSS exist
+      const themes = Object.keys(pluginThemes).reduce((valid, themeName) => {
+        const config = pluginThemes[themeName]
+
+        if (isBoolean(config.darkMode) && isString(config.cssPath)) {
```

Comment:
It will only be imported if I set `darkMode` to `true`. I did not get it, this property is useful for what?

Reply:
Nop, @luciorubeens, it only checks that `config.darkMode` is boolean. That property is used could be used to allow a light and a dark variant of each themes, but currently it's used to decide which style should use Changelly.